### PR TITLE
Fix installation commands

### PR DESCRIPTION
**Pull request by RONTheCookie**

### CODE REVIEW

**Comment by jellz** [comment]
You should change "cmds" to "commands" in the PR title 😄

File: README.md [hunk 1]
```diff
@@ -52,11 +52,9 @@ Or as root:
 ```shell
 # curl -sSL https://raw.githubusercontent.com/IonLanguage/Ion.CLI/master/install.sh | sh
 ```
-If you wish to view the source code of the installer script, simply navigate to its source URL:
+If you wish to view the source code of the installer script, simply navigate to its [source URL](https://raw.githubusercontent.com/IonLanguage/Ion.CLI/master/install.sh):
```
Comment (cloudrex) [inline README.md]: Remove the ":" character at the end and replace it with a dot.

File: README.md [hunk 2]
```diff
@@ -45,9 +45,13 @@ Head over to the releases page on the CLI utility's repository to get your insta
 Get started quickly and conviniently with the following one-liner:
 
 ```shell
-$ curl -sSL https://raw.githubusercontent.com/IonLanguage/Ion.CLI/master/install.sh | sh
+$ curl -sSL https://raw.githubusercontent.com/IonLanguage/Ion.CLI/master/install.sh | sudo sh
 ```
 
+Or as root:
```
Comment (cloudrex) [inline README.md]: No need to specify root instructions, a normal Linux user should be able to determine such on their own.

File: README.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ Head over to the releases page on the CLI utility's repository to get your insta
 Get started quickly and conviniently with the following one-liner:
 
 ```shell
-$ curl -sSL https://raw.githubusercontent.com/IonLanguage/Ion.CLI/master/install.sh | bash
+$ curl -sSL https://raw.githubusercontent.com/IonLanguage/Ion.CLI/master/install.sh | sh
```
Comment (cloudrex) [inline README.md]: I've skimmed through the internet for some quick research and it appears `sh` is supported more than `bash` so we're going with it.

**Pull request merged**